RELEASE.md

```diff
@@ -17,6 +17,7 @@ Please follow the established format:
 - Upgrade to `dart-scss` and update the recommended Node version to v16. (#1026)
 - Fix the export functionality of experiment tracking data. (#1033)
 - Color cleanup and refactor. (#1007)
+- Lazy loading for the experiment tracking. (#1041)
```

Comment:
```suggestion
- Add lazy loading to experiment tracking. (#1041)
```

Can you also remove the line above please? This one:

```
- Color cleanup and refactor. (#1007)
```

We don't need to add that because it's not something a user would see or notice.

Reply:
oh do we not need to mention it? if so i can remove it 👍 thanks for letting me know

Reply:
No we don't need to mention the color work in the release notes. Only things that would be noticeable to the user or allow them to change the way they interact with Viz (a new Node version, for example) should go in there.

cc @cvetankanechevska